<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SampleApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLayout {margin: 0px}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="customMenuWidget" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	color: black;
	box-shadow: 0px -3px 5px #a6a6a6;
}
QWidget#customMenuWidget { background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.458128 rgba(109, 207, 255, 255), stop:1 rgba(57, 127, 158, 255))}</string>
      </property>
      <widget class="QPushButton" name="lightingBtn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Lighting</string>
       </property>
      </widget>
      <widget class="QPushButton" name="settingsBtn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
      <widget class="QPushButton" name="comfortBtn">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Comfort</string>
       </property>
      </widget>
      <widget class="QPushButton" name="securityBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Security</string>
       </property>
      </widget>
      <widget class="QLabel" name="control4Lbl">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="myresources.qrc">:/new/image/resources/Control4logo.png</pixmap>
       </property>
      </widget>
      <widget class="QPushButton" name="watchBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Watch</string>
       </property>
      </widget>
      <widget class="QPushButton" name="appsBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Apps</string>
       </property>
      </widget>
      <widget class="QPushButton" name="shadesBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Shades</string>
       </property>
      </widget>
      <widget class="QPushButton" name="listenBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>Listen</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
